src/main/resources/view/MainWindow.fxml

```diff
@@ -14,7 +14,7 @@
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
          minWidth="450" minHeight="600">
   <icons>
-    <Image url="@/images/address_book_32.png" />
+    <Image url="@/images/progress_checker_32.png" />
```

Comment:
Might want to change this back to ```address_book_32.png``` for the image file to be valid. P.S. Not recommended to rename the image file to solve this instead because the image does not belong to us so we shouldn't claim it.